src/styles/utils/variables.scss

```
@@ -0,0 +1,7 @@
+// COLORS
+$header-hover-color: #00acdc;
+$white-accent: #FFF;
+$main-text-color: #060B35;
+$secondary-text-color: #616070;
+$blue-accent: #00ACDC;
+$width-content: 166px;
```

No need to hardcode width for the content
```suggestion
$width-content: 100%;
```